src/examples/simple_zkapp.ts

```diff
@@ -77,7 +77,9 @@ let zkappKey = PrivateKey.random();
 let zkappAddress = zkappKey.toPublicKey();
 
 // a special account that is allowed to pull out half of the zkapp balance, once
-let privilegedKey = PrivateKey.random();
+let privilegedKey = PrivateKey.fromBase58(
+  'EKEeoESE2A41YQnSht9f7mjiKpJSeZ4jnfHXYatYi8xJdYSxWBep'
+);
```

Comment:
Huh, why is that? :thinking:

Reply:
Constants that are used in constraints are baked into the circuit digest as well as prover and verifier keys. This example was using `PrivateKey.random()` so the circuit was different every time. Since we compare the circuit digest to decide whether or not we can use a cached key, the answer was "no" every time.

Reply:
Ah, yes, pretty straightforward. That makes sense, thank you!